src/backend/distributed/transaction/relation_access_tracking.c

```diff
@@ -120,6 +122,18 @@ static bool HoldsConflictingLockWithReferencedRelations(Oid relationId,
 														conflictingAccessMode);
 
 
+/*
+ * InitRelationAccessHash performs initialization of the
+ * infrastructure in this file at backend start.
+ */
+void
+InitRelationAccessHash(void)
+{
+	/* (relationId) = [relationAccessMode] hash */
+	AllocateRelationAccessHash();
```

Comment:
May be rename  AllocateRelationAccessHash() to InitRelationAccessHash() and remove the wrapper?

Reply:
We have bunch of `InitXXX` APIs, so I have a slight preference to keep `InitRelationAccessHash` just for readability of this part: https://github.com/citusdata/citus/blob/0f50bef696e8925374a7423f89b7513bf6f31d44/src/backend/distributed/shared_library_init.c#L389-L395

Reply:
I agree, it's better to keep it uniform. My comment was more on the lines, remove the function `AllocateRelationAccessHash`() and put the code directly in `InitRelationAccessHash` , since this is not a hot path, I guess that's fine.